src/ui/components/Button/index.js

```diff
@@ -51,7 +58,9 @@ export default class Button extends React.Component {
       to,
       ...rest
     } = this.props;
-    const props = { ...rest };
+    const props: Object = { ...rest };
```

Comment:
I am not sure if Object is ok here or if I should create up something like "internalProps". Initially I went that route but it wasn't working so with buttonType. if that's preferred tho, I will look into that more..

Reply:
We can use `$Rest` I think.